dcos/config.py

```diff
@@ -213,6 +213,26 @@ def get_config_schema(command):
     return subcommand.config_schema(executable, command)
 
 
+def get_property_description(name, value):
```

Comment:
The rest of the public APIs takes a `name` that is a dot separated path. E.g. `unset` and `set_value`. Would be nice to keep that consistent. For example we can call `split_key` from here.